test/models/user_test.rb

```diff
@@ -0,0 +1,13 @@
+require 'test_helper'
+
+class UserTest < ActiveSupport::TestCase
+
+  def setup
+    @user = User.new(first_name: "Lade", last_name: " ")
+  end
+
+  test "Should be valid" do
```

Comment:
This test is vague and contradicting itself. You are saying "should be valid" but you are asserting for user not to be valid.